services/121-service/src/notifications/message-incoming/message-incoming.service.ts

```diff
@@ -418,12 +420,13 @@ export class MessageIncomingService {
 
     const registrationsWithOpenVouchers =
       await this.getRegistrationsWithOpenVouchers(registrationsWithPhoneNumber);
-
     // If no registrations with outstanding vouchers or messages: send auto-reply
     if (
       registrationsWithOpenVouchers.length === 0 &&
       registrationsWithPendingMessage.length === 0
     ) {
+      // Hardcoded for PA-triggered auto-response, as this method isn't user-initiated and lacks userId.
+      const userId = 1;
```

Comment:
I think you should find the userId by admin and then set the userId like we did in migration.

Reply:
You are right, let me fix it.

Reply:
Done